### PR TITLE
feat(dedicated): add tracking on byoip slice and aggregate

### PR DESCRIPTION
**Pull request by benjaminlepine**
ref: MANAGER-12733

| Question         | Answer
| ---------------- | ---
| Branch?          | feat/MANAGER-12381
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-12381
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/ip/components/list/byoip/aggregate/ip-ip-byoip-aggregate.controller.js [hunk 1]
```diff
@@ -10,6 +11,8 @@ export default class IpByoipAggregateController {
     this.errorMessage = null;
     this.Alerter = Alerter;
     this.$translate = $translate;
+    this.atInternet = atInternet;
+    atInternet.trackPage({ name: TRACKING_PREFIX_AGGREGATE });
```
Comment (darsene) [inline packages/manager/apps/dedicated/client/app/ip/components/list/byoip/aggregate/ip-ip-byoip-aggregate.controller.js:15]: sounds more logic to be move on `$onInit` function

File: packages/manager/apps/dedicated/client/app/ip/components/list/byoip/aggregate/ip-ip-byoip-aggregate.controller.js [hunk 2]
```diff
@@ -29,10 +32,18 @@ export default class IpByoipAggregateController {
   }
 
   cancelAggregate() {
+    this.atInternet.trackClick({
+      type: 'action',
+      name: TRACKING_PREFIX_AGGREGATE + '::cancel',
+    });
```
Comment (darsene) [inline packages/manager/apps/dedicated/client/app/ip/components/list/byoip/aggregate/ip-ip-byoip-aggregate.controller.js:38]: As you're repeating some code, create a function for `trackClick`

```
function trackClick(hit) {
 this.atInternet.trackClick({
      type: 'action',
      name: `${TRACKING_PREFIX_AGGREGATE}::${hit}`,
    });
}
```

Comment (darsene) [inline packages/manager/apps/dedicated/client/app/ip/components/list/byoip/aggregate/ip-ip-byoip-aggregate.controller.js:38]:
```suggestion
    this.trackClick('cancel');
```

File: packages/manager/apps/dedicated/client/app/ip/components/list/byoip/aggregate/ip-ip-byoip-aggregate.controller.js [hunk 3]
```diff
@@ -29,10 +32,18 @@ export default class IpByoipAggregateController {
...
     this.$scope.resetAction();
   }
 
   aggregate() {
+    this.atInternet.trackClick({
+      type: 'action',
+      name: TRACKING_PREFIX_AGGREGATE + '::confirm',
+    });
```
Comment (darsene) [inline packages/manager/apps/dedicated/client/app/ip/components/list/byoip/aggregate/ip-ip-byoip-aggregate.controller.js:46]:
```suggestion
   this.trackClick('confirm');
```

File: packages/manager/apps/dedicated/client/app/ip/components/list/byoip/constants.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+export const TRACKING_PREFIX = 'dedicated::ip';
+export const TRACKING_PREFIX_SLICE = 'dedicated::ip::slice-ip-block';
+export const TRACKING_PREFIX_AGGREGATE = 'dedicated::ip::merge-ip-block';
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/components/list/byoip/constants.js:3]: **nitpick:** 
```suggestion
export const TRACKING_PREFIX = 'dedicated::ip';
export const TRACKING_PREFIX_SLICE = `${TRACKING_PREFIX}::slice-ip-block`;
export const TRACKING_PREFIX_AGGREGATE = `${TRACKING_PREFIX}::merge-ip-block`;
```

File: packages/manager/apps/dedicated/client/app/ip/components/list/byoip/slice/ip-ip-byoip-slice.controller.js [hunk 5]
```diff
@@ -10,6 +12,8 @@ export default class IpByoipSliceController {
     this.errorMessage = null;
     this.Alerter = Alerter;
     this.$translate = $translate;
+    this.atInternet = atInternet;
+    atInternet.trackPage({ name: TRACKING_PREFIX_SLICE });
```
Comment (darsene) [inline packages/manager/apps/dedicated/client/app/ip/components/list/byoip/slice/ip-ip-byoip-slice.controller.js:16]: same than https://github.com/ovh/manager/pull/10994#discussion_r1498915949

File: packages/manager/apps/dedicated/client/app/ip/components/list/byoip/slice/ip-ip-byoip-slice.controller.js [hunk 6]
```diff
@@ -30,10 +34,18 @@ export default class IpByoipSliceController {
   }
 
   cancelSlice() {
+    this.atInternet.trackClick({
+      type: 'action',
+      name: TRACKING_PREFIX_SLICE + '::cancel',
+    });
```
Comment (darsene) [inline packages/manager/apps/dedicated/client/app/ip/components/list/byoip/slice/ip-ip-byoip-slice.controller.js:40]: same than https://github.com/ovh/manager/pull/10994/files#r1498920411

File: packages/manager/apps/dedicated/client/app/ip/components/list/byoip/slice/ip-ip-byoip-slice.controller.js [hunk 7]
```diff
@@ -30,10 +34,18 @@ export default class IpByoipSliceController {
...
     this.$scope.resetAction();
   }
 
   slice() {
+    this.atInternet.trackClick({
+      type: 'action',
+      name: TRACKING_PREFIX_SLICE + '::confirm',
+    });
```
Comment (darsene) [inline packages/manager/apps/dedicated/client/app/ip/components/list/byoip/slice/ip-ip-byoip-slice.controller.js:48]: same than https://github.com/ovh/manager/pull/10994/files#r1498921783

**Review by anooparveti — APPROVED** [review]

**Review by qpavy — APPROVED** [review]

**Pull request merged**